packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/create/module.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+
+import component from './component';
+import routing from './routing';
+import service from '../service';
+
+const moduleName =
+  'ovhManagerOctaviaLoadBalancerPoolsDetailHealthMonitorCreate';
+
+angular
+  .module(moduleName, [])
+  .config(routing)
+  .component('octaviaLoadBalancerPoolsDetailHealthMonitorCreate', component)
+  .service('OctaviaLoadBalancerHealthMonitorService', service)
```

Comment:
I'm not sure you need to redeclare the service, which is already declared in the parent.